cmd/gke-gcloud-auth-plugin/cred.go

```diff
@@ -89,7 +93,8 @@ func newPlugin(tokenProvider tokenProvider) *plugin {
 var (
 	useApplicationDefaultCredentials = pflag.Bool("use_application_default_credentials", false, "Output is an ExecCredential filled with application default credentials.")
 	useEdgeCloud                     = pflag.Bool("use_edge_cloud", false, "Output is an ExecCredential for an Edge Cloud cluster.")
-	project                          = pflag.String("project", "", fmt.Sprintf("Parent project of the Cluster %s.", useEdgeCloudOnlyMessage))
+	project                          = pflag.String("project", "", fmt.Sprintf("Parent project of the Cluster %s.", useEdgeCloudOnlyRequiredMessage))
+	account                          = pflag.String("account", "", "Optional account to use for gcloud config command")
 	location                         = pflag.String("location", "", fmt.Sprintf("Location of the Cluster %s.", useEdgeCloudOnlyMessage))
 	cluster                          = pflag.String("cluster", "", fmt.Sprintf("Name of the Cluster %s.", useEdgeCloudOnlyMessage))
 	impersonateServiceAccount        = pflag.String("impersonate_service_account", "", fmt.Sprintf("Impersonate a service account to retrieve tokens for the Cluster %s.", useEdgeCloudOnlyMessage))
```

Comment:
I think we can remove "useEdgeCloudOnlyRequiredMessage" or "useEdgeCloudOnlyMessage" as we can use it in multiple places, right? Having that "edge cloud" only is not really relevant..

Reply:
Touched on it in the above comment, this flag is applicable to both, but it is required when using edge_cloud, so that is being noted here, however the error message when not provided is clear about why it failed, so the user would easily see after it fails that they need to provide it. 

I prefer the UX of knowing it is required ahead of time rather than running the command and then seeing it fail and fixing it based on the error message. If that's not a pattern we want here I wont die on hill, maybe listing out the required flags in the desc of `useEdgeCloud` is a good middle ground

Reply:
Replied on other comment